internal/engine/experiment/tor/tor_test.go

```diff
@@ -292,8 +291,9 @@ func TestResultsCollectorMeasureSingleTargetWithFailure(t *testing.T) {
 		new(model.Measurement),
 		model.NewPrinterCallbacks(log.Log),
 	)
-	rc.flexibleConnect = func(context.Context, keytarget) (oonitemplates.Results, error) {
-		return oonitemplates.Results{}, errors.New("mocked error")
+	rc.flexibleConnect = func(context.Context, keytarget) (*measurex.ArchivalMeasurement, *string) {
+		failure := "mocked error"
+		return &measurex.ArchivalMeasurement{}, &failure
```

Comment:
Here I needed to adapt `flexibleConnect` (remember: it always returns a non-nil measurement)